src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
It's set up this way because I would like there to be a mode where RandomPolicy is the only policy used, and I can control the seed. The missing functions are:
```
compInlineRandomStress == compStressCompile(STRESS_RANDOM_INLINE, 100)
compInlineStress = compStressCompille(STRESS_INLINE, 50)
```
Thus by setting things as follows I can get the mode I want:
```
set complus_jitstress=2020202
set complus_jitstressModeNames=STRESS_RANDOM_INLINE
set complus_jitstressModeNamesNot=STRESS_INLINE
```
(note setting `jitStressModeNames` does not exclude other modes from being active, which seems a little odd).